packages/sourcecred/src/ui/components/LedgerViewer/LedgerViewer.js

```diff
@@ -82,7 +86,17 @@ export const LedgerViewer = ({
     setAllocation(null);
   }, []);
 
+  const handleStartDateChange = (date) => {
+    setStartDateFilter(date);
+  };
+  const handleEndDateChange = (date) => {
+    setEndDateFilter(date);
+  };
+
   const eventLog = useMemo(() => [...ledger.eventLog()], [ledger]);
+  const [startDateFilter, setStartDateFilter] = useState(new Date());
+  const [endDateFilter, setEndDateFilter] = useState(new Date());
+
```

Comment:
Could you use the new Date placeholder outside the component so that it's not evaluated at every re-render? a single `new Date()`

Reply:
sure that sounds reasonable however the initial state will be changed later to be empty this was just for testing

Reply:
sounds reasonable